bigtable/cmd/cbt/cbt.go

```diff
@@ -1018,8 +1157,17 @@ func printRow(r bigtable.Row) {
 		sort.Sort(byColumn(ris))
 		for _, ri := range ris {
 			ts := time.Unix(0, int64(ri.Timestamp)*1e3)
-			fmt.Printf("  %-40s @ %s\n", ri.Column, ts.Format("2006/01/02-15:04:05.000000"))
-			fmt.Printf("    %q\n", ri.Value)
+			fmt.Printf("  %-40s @ %s\n",
```

Comment:
Nope. Long source lines bug me. :) I just wrapped the arguments.  I didn't change semantics of this line.

If you mean the overall change, still Nope.  In fact, the first test I wrote passed without the change -- and after.

Reply:
Great!

Reply:
`format` doesn't print anything.  It just computes the string that should be printed with `fmt.Print(formatted)` below.

It obviously replaces `%q`, which is the whole point of this PR.  It also has to handle indentation, previously provided by the spaces before `%q` in `"    %q\n"`, because formatted values may be multi-line.